src/main/java/seedu/address/commons/core/GuiSettings.java

```diff
@@ -60,20 +67,22 @@ public boolean equals(Object other) {
 
         return windowWidth == o.windowWidth
                 && windowHeight == o.windowHeight
-                && Objects.equals(windowCoordinates, o.windowCoordinates);
+                && Objects.equals(windowCoordinates, o.windowCoordinates)
+                && attributeOrder.equals(o.attributeOrder);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(windowWidth, windowHeight, windowCoordinates);
+        return Objects.hash(windowWidth, windowHeight, windowCoordinates, attributeOrder);
     }
 
     @Override
     public String toString() {
         StringBuilder sb = new StringBuilder();
         sb.append("Width : " + windowWidth + "\n");
         sb.append("Height : " + windowHeight + "\n");
-        sb.append("Position : " + windowCoordinates);
+        sb.append("Position : " + windowCoordinates + "\n");
+        sb.append("Attribute Order : " + attributeOrder);
```

Comment:
Just to make sure, no ` + "\n"` after `attributeOrder` because it is in the end position?

Reply:
Yes. The line removed did not have `+ "\n"`, so I just followed accordingly.